launch/tier4_localization_launch/launch/localization.launch.xml

```diff
@@ -13,12 +14,26 @@
     <group>
       <push-ros-namespace namespace="util"/>
       <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.xml">
-        <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
+        <arg name="input/pointcloud" value="$(var input/origin_pointcloud)"/>
         <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
         <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
         <arg name="tier4_localization_launch_param_path" value="$(var tier4_localization_launch_param_path)"/>
       </include>
     </group>
+    <!-- util module -->
+    <group>
+      <push-ros-namespace namespace="util_no_ground"/>
+      <include file="$(find-pkg-share tier4_localization_launch)/launch/util/util.launch.xml">
+        <arg name="input/pointcloud" value="$(var input/no_ground_pointcloud)"/>
+        <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
+        <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
+        <arg name="tier4_localization_launch_param_path" value="$(var tier4_localization_launch_param_path)"/>
+      </include>
+    </group>
+    <!-- remove ground module -->
+    <include file="$(find-pkg-share ground_segmentation)/launch/scan_ground_filter.launch.py">
+      <arg name="input/pointcloud" value="$(var input/orign_pointcloud)"/>
```

Comment:
This `orign_pointcloud` is a mistake for `origin_pointcloud`, right?

Reply:
Thanks